gradle/libs.versions.toml

```diff
@@ -37,7 +37,7 @@ swagger-codegen-plugin = { module = "gradle.plugin.org.hidetake:gradle-swagger-g
 # we need to manually upgrade here in order to have the library compatible with jackson
 swagger-jersey2-jaxrs = { module = "io.swagger:swagger-jersey2-jaxrs", version = { strictly = "1.6.2" } }
 
-springdoc-openapi-plugin = { module = "org.springdoc:springdoc-openapi-gradle-plugin", version = "1.7.0" }
+springdoc-openapi-plugin = { module = "org.springdoc:springdoc-openapi-gradle-plugin", version = "1.8.0" }
```

Comment:
Since you updated this plugin the method `makeOpenApiTaskReactive` in SpringDocOpenApiConfigurePlugin.kt#53 should no longer be necessary, since that should now be done correctly by this plugin itself.
https://github.com/springdoc/springdoc-openapi-gradle-plugin/pull/124

Reply:
done